CONTRIBUTING.md

```diff
@@ -1,60 +1,139 @@
+The `untriaged` label is assigned automatically. During the triage process, the documentation team will add the appropriate labels, assign the issue to a technical writer, and prioritize the request. We may follow up with you for additional information. 
 
-**Create an Issue**
+## Finding contributions to work on
 
-Use the Github issue tracker to describe the change you'd like to make. 
+If you’d like to contribute but don't know where to start, try browsing existing [issues](https://github.com/opensearch-project/documentation-website/issues). Our projects use custom GitHub issue labels for status, version, type of request, and so on, but we recommend looking at any issues labeled `good first issue` first. 
 
-1. Go to https://github.com/opensearch-project/documentation-website/issues and select *New issue*.
-2. Enter the requested information and include as much detail as possible, especially which version or versions the request affects.
-3. Select *Submit new issue*. 
+## Contributing content
 
+There are two ways to contribute content depending on the magnitude of the change:
 
-The ‘untriaged’ label is assigned automatically. During the triage process, the documentation team will add the appropriate labels, assign the issue to a technical writer, and prioritize the request. We may follow up with you for additional information. 
+- [Minor changes](#making-minor-changes): For small changes like fixing a typo or adding a parameter, you can edit files in GitHub directly. This approach does not require cloning the repository and does not let you test the documentation.
+- [Major changes](#making-major-changes): For changes you want to test first, like reorganizing pages or adding a table or section, you can edit files locally and push the changes to GitHub. This approach requires setting up a local version of the repository and lets you test documentation.
 
+### Documentation workflow
 
-**Create a pull request**
+The workflow for contributing documentation is no different than the one for contributing code:
 
-If you’d like to make a change directly to the content, create a pull request. If it’s a quick fix, we should be able to release the update quickly. Bigger requests might take a bit of time for us to review. 
+- Make your changes
+- Build locally to check your work (only possible if you are making changes locally)
+- Submit a [pull request](https://github.com/opensearch-project/documentation-website/pulls) (PR)
+- Maintainers review and merge your PR
 
-Note that a pull request requires DCO sign off before we can merge it. You can use the -s command line option to append this automatically to your commit message, for example $ git commit -s -m 'This is my commit message'. For more information, see https://github.com/apps/dco.
+If the change requires significant work, open an issue where we can first discuss your request.
```

Comment:
Not sure if this is needed since we already included reorganizing pages in a previous section.